airgun/entities/os.py

```diff
@@ -12,6 +12,11 @@ def create(self, values):
         view.fill(values)
         view.submit.click()
 
+    def delete_operating_system(self, value):
```

Comment:
I had done some naming refactoring yesterday, such long name doesn't make sense as the method is a part of operating system entity, obviously `delete` operation will delete operating system, not something else. Please rename it to simple `delete`.